utils/obsproc/superob.h

```diff
@@ -8,11 +8,47 @@
 
 namespace gdasapp {
   namespace superobutils {
+    // Function to compute distance between two points
+    struct GeoPoint {
+        double latitude;
+        double longitude;
+    };
+    const double EARTH_RADIUS_KM = 6371.0;
+
+    // Function to convert degrees to radians
+    double toRadians(double degree) {
+        return degree * (M_PI / 180.0);
+    }
+
+    // Function to calculate the Haversine distance between two points
```

Comment:
You should leverage functions from `oops` and/or `atlas` for this. 
Look in the `atlas` source code for `Earth::distance`.

Reply:
Distance function from atlas is used in the updated code.